apps/svelte.dev/content/blog/2024-11-01-whats-new-in-svelte-november-2024.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "What's new in Svelte: November 2024"
+description: 'New features in Svelte 5 and a ton of new community libraries'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+In case you missed the [announcement post](https://svelte.dev/blog/svelte-5-is-alive), the brand new [omnisite](https://svelte.dev/blog/the-omnisite) or the maintainers [shipping it LIVE during Svelte Summit](https://www.youtube.com/live/fAPFsRP-mbc?si=Atd1WNpoMddssSna&t=13084)... Svelte 5 is out!
+
+Also announced at Svelte Summit was [SvelteHack 2024: A Svelte Hackathon (with not-so-svelte prizes)](https://hack.sveltesociety.dev/2024). The submission period ends January 9 and only projects started after October 18th are eligible. So get those `$`s ready and start hacking!
+
+Since the release of Svelte 5, there's already been one minor version to introduce a couple of helpful features. We'll dive into those - along with a number of updates across the CLI, SvelteKit and more - in this month's roundup...
+
+## What's new in Svelte and the Svelte CLI
```

Comment:
Since the CLI is mostly a CLI for creating SvelteKit projects, it might make more sense to group it with SvelteKit rather than Svelte core